File: Indixpert-Sept-batch2025/Python/tuple/task6.py
list=[]
for i in range(0,2):
    a = int(input("Enter Roll Number = "))
    b = str(input("Enter Name = "))
    c = input("Enter Class = ")
    d = str(input("Enter Section = "))
    student = (a,b,c,d)
    list.append(student)
t=tuple(list)    

print(t)

File: Indixpert-Sept-batch2025/C Languege/structure/task6.c
#include <stdio.h>
struct student
{
    char name[20];
    int rollno;
};
struct student getdata(struct student);   // int getdata(int);
struct student getdata(struct student ob) // int getdata(int num);
{
    printf("enter name=");
    gets(ob.name);
    printf("enter roll number=");
    scanf("%d", &ob.rollno);
    getchar();
    return ob;
}
void show(struct student obj)
{
    printf("%s\n", obj.name);
    printf("%d\n", obj.rollno);
}

void main()
{
    struct student obb;
    obb = getdata(obb);
    show(obb);
}
